tools/testsys/src/run.rs

```diff
@@ -37,13 +40,26 @@ pub(crate) struct Run {
     #[clap(long, env = "TESTSYS_TEST_CONFIG_PATH", parse(from_os_str))]
     test_config_path: PathBuf,
 
+    /// The path to the EKS-A management cluster for vSphere K8s cluster creation
+    #[clap(long, env = "TESTSYS_MGMT_CLUSTER_KUBECONFIG", parse(from_os_str))]
+    mgmt_cluster_kubeconfig: PathBuf,
+
     /// Use this named repo infrastructure from Infra.toml for upgrade/downgrade testing.
     #[clap(long, env = "PUBLISH_REPO")]
     repo: Option<String>,
 
+    /// The name of the vSphere data center in `Infra.toml` that should be used for testing
+    /// If no data center is provided, the first one in `vmware.datacenters` will be used
```

Comment:
Was `vmware.datacenters` defined in a different PR?

Reply:
It's already a part of `pubsys_config`